src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -115,6 +117,9 @@ public function putOffExpiration(Key $key, $ttl)
 
                 $store->putOffExpiration($key, $adjustedTtl);
                 ++$successCount;
+            } catch (NotExpirableStoreException $e) {
+                $this->logger->notice('This store does not support expiration of locks.', $store);
+                ++$notExpirableStoresCount;
```

Comment:
Should be take into account in`$this->strategy->isMet($successCount, $storesCount)`

Reply:
I have now removed throwing exceptions and added new `ExpirableStoreInterface` instead so reverting this change.